src/Compilers/Core/Portable/Emit/NoPia/CommonEmbeddedMethod.cs

```diff
@@ -68,6 +68,7 @@ internal abstract class CommonEmbeddedMethod : CommonEmbeddedMember<TMethodSymbo
             protected abstract bool AcceptsExtraArguments { get; }
             protected abstract Cci.ISignature UnderlyingMethodSignature { get; }
             protected abstract Cci.INamespace ContainingNamespace { get; }
+            public abstract bool HasBody { get; }
```

Comment:
>public abstract bool HasBody { get; } [](start = 12, length = 37)

It is probably not worth introducing this property. Embedded NpPia methods never have bodies. We can simply adjust code while taking this into account